modules/aggregation-layers/src/hexagon-layer/hexagon-cell-layer-vertex.glsl.ts

```diff
@@ -0,0 +1,72 @@
+import {getHexbinCentroidGlsl} from './hexbin';
```

Comment:
Nit: My convention is usually to not decapitalize in these cases. I know kepler does it, not sure about deck.
```suggestion
import {getHexbinCentroidGLSL} from './hexbin';
```